nefertari/elasticsearch.py

```diff
@@ -245,6 +242,74 @@ def get_document_proxies_by_type(cls, doc_type):
         raise UnknownDocumentProxiesTypeError('You have no proxy for this %s document type' % doc_type)
 
 
+class BoundedTransaction(type):
```

Comment:
*Bounded* means that the result set is finite (it has bounds).

Do you mean to say *bound*, past particle of the verb *to bind*?

See wikipedia: https://en.wikipedia.org/wiki/Bounded_set